proprietary/Components/src/denhaag/link-group.tokens.json

```diff
@@ -34,10 +34,11 @@
         "padding-block-end": { "value": "0" }
       },
       "denhaag-link-icon": {
-        "font-size": { "value": "0.75rem" }
+        "font-size": { "value": "0.75rem" },
+        "align-self": { "value": "start" },
+        "flex": { "value": "1 0 auto" }
```

Comment:
Do we really need these two values to be customisable through design tokens?

Reply:
@robbieatacato , thought we discussed this would be applied on the denhaag-link instead of the link-group since all other elements require the denhaag-link component. 

@bddjong , the properties robbie added fixes the wierd icon alignment, where gap / icon width is reduced.

Reply:
Can these properties not just be added to the css? Bit strange to require a layout fix to be added to the theme definition.

Reply:
For the "flex: 1 0 auto;" I agree with you.

The alignment of the icons is in a lot of cases 'flex-start', but in several cases it must be 'center'. Therefor a CSS variable is in my opinion a good practice.

Reply:
I restored the tokens + scss.
Only left changes: the bem file with added `class="denhaag-link__label"` tot the span holding the text as the link component.

I wil move the styling changes to the link component.